docker/docker-compose.local.yml

```diff
@@ -101,7 +101,7 @@ services:
   # For testing.
   test:
     container_name: test
-    image: clerk:latest
+    image: anikalaw/clerk:${TAG}
```

Comment:
we probably don't want to set this for docker-compose.local.yml because it is also used for local testing
perhaps create a new docker-compose.ci.yml and set the image to  anikalaw/clerk:${TAG} in that one?

then you can reference docker-compose.ci.yml in the GitHub workflows

Reply:
ah yes ... that rings a bell: we are using the docker-compose.local test container when we type `make test`... I think I ought to create spreadsheet representing the different images used for the containers in the four docker-compose files and send them to you